src/bridge/cockpitpackages.c

```diff
@@ -400,20 +418,14 @@ read_package_manifest (const gchar *directory,
     }
   else
     {
-      override = read_json_file (directory, "override.json", &error);
-      if (error)
-        {
-          if (g_error_matches (error, G_FILE_ERROR, G_FILE_ERROR_NOENT))
-            g_debug ("%s: no override found", package);
-          else
-            g_message ("%s: couldn't read override.json: %s", package, error->message);
-          g_clear_error (&error);
-        }
-      if (override)
-        {
-          cockpit_json_patch (manifest, override);
-          json_object_unref (override);
-        }
+      /* possible override locations, in ascending priority */
+      apply_override (manifest, directory, "override.json");
+
+      g_autofree gchar *package_override_name = g_strconcat (package, ".override.json", NULL);
+      apply_override (manifest, PACKAGE_SYSCONF_DIR "/cockpit", package_override_name);
```

Comment:
We need to talk about this a bit, and maybe do a big cleanup: cockpitconf tries also to check the XDG_CONFIG_DIRS variable.  I'm okay with killing that off, for what it's worth, but we should at least be consistent.  There's even a function in there that returns the list of config directories.

Reply:
Good point, thanks! Changed to use `cockpit_conf_get_dirs()`.